src/platform-includes/distributed-tracing/how-to-use/javascript.astro.mdx

```diff
@@ -0,0 +1,14 @@
+If you're using our Astro SDK, distributed tracing will work out of the box for the client and server runtimes.
+To get around possible [Browser CORS](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS) issues, you should define `tracePropagationTargets` on the client-side.
+
+```js
+// sentry.client.config.js
+Sentry.init({
+  dsn: "___PUBLIC_DSN___",
+  integrations: [new BrowserTracing()],
+  tracePropagationTargets: [
+    "https://myproject.org",
+    "https://.*.otherservice.org/.*",
```

Comment:
h: Do we actually support globbing here? I thought we only do regexp.match() or naive string.contains().

Reply:
Hmm probably not the best example, let me come up with a better one (copy 🍝 so maybe also relevant for other pages)

`.*` is valid regex syntax though 🤔

Reply:
Addressed this for Astro in [7d18a09](https://github.com/getsentry/sentry-docs/pull/8267/commits/7d18a09e6c43fb30adfb2a5aa21f03314e793443) but we have the same invalid string in a bunch of places. Will open a separate PR to fix there. Also asked mobile folks because the same string is used in flutter and apple docs.